index.html

```diff
@@ -71,24 +71,25 @@
 
 ```ABNF
   Server-Timing        = #server-timing-metric
-  server-timing-metric = metric  [ OWS ";" OWS description ]
-  metric               = metric-name [ OWS "=" OWS metric-value ]
+  server-timing-metric = metric-name *( OWS ";" OWS server-timing-param )
   metric-name          = token
-  metric-value         = *DIGIT [ "." 1*DIGIT ]
-  description          = token | quoted-string
+  server-timing-param  = token OWS [ "=" OWS ( token / quoted-string ) ]
```

Comment:
It's better to break apart `server-timing-param` into `server-timing-param-name OWS [ "=" OWS server-timing-param-value ]` where `server-timing-param-name = token` and `server-timing-param-value = ( token / quoted-string)`.
That way you can later reference them directly.